xonsh/completers/tools.py

```diff
@@ -201,3 +203,35 @@ def apply_lprefix(comps, lprefix):
                 yield comp
         else:
             yield RichCompletion(comp, prefix_len=lprefix)
+
+
+def bash_complete_completer(ctx: CommandContext, **env: str):
```

Comment:
should this be decorated as a `contextual` completer?

Reply:
I should not be decorated. it is just a helper function and not intended to be called directly as a completer.